core/tests/test_utils.py

```diff
@@ -319,9 +369,22 @@ def set_moderators(self, moderator_usernames):
             config_domain.MODERATOR_USERNAMES, moderator_usernames)
 
     def get_current_logged_in_user_id(self):
+        """Gets the user_id of the current logged-in user.
+
+        Returns:
+            str. Value of USER_ID env variable.
+        """
         return os.environ['USER_ID']
 
     def get_user_id_from_email(self, email):
+        """Gets the user_id corresponding to the given email.
+
+        Args:
+            email: str. valid email id stored in the Appengine database.
+
+        Returns:
+            user_id: str. Id of user possessing given email.
+        """
         return current_user_services.get_user_id_from_email(email)
 
     def save_new_default_exploration(
```

Comment:
Docstrings for args? Please check existing docstrings to ensure they're complete.